opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -80,6 +80,12 @@ struct StackdriverOptions {
   // instead. Useful for testing.
   std::unique_ptr<google::monitoring::v3::MetricService::StubInterface>
       metric_service_stub;
+
+  // Optional: A function which can be used to modify the grpc::ClientContext
+  // before making API requests.
+  // This can be used to add custom credentials for example.
```

Comment:
Please add:
This function will be called by the exporter before every CreateTimeSeries and CreateMetricDescriptor RPC.
This function will not be called in parallel.

Reply:
Done